indra/sources/medscan/api.py

```diff
@@ -48,14 +136,25 @@ def process_file(filename, medscan_resource_dir, num_documents=None):
     num_documents : int
         The number of documents to process, or None to process all of the
         documents within the csxml file.
+    progress_updates: bool
+        Whether to print progress updates to the console
```

Comment:
The typical solution for this is to change the logging level of the logger for this module. If a user wants to suppress logging here, they can set the level to logging.WARNING, for instance. So I don't think this needs to be an API argument.